README.md

```diff
@@ -178,14 +178,14 @@ are in the file named "LICENSE.apache20" in this directory.
 [dependabot]:https://dependabot.com
 [dependabot img]:https://api.dependabot.com/badges/status?host=github&repo=checkstyle/checkstyle
 
-[closed issues]:https://github.com/checkstyle/checkstyle/actions/workflows/no_old_refs.yml
-[closed issues img]:https://github.com/checkstyle/checkstyle/actions/workflows/no_old_refs.yml/badge.svg
+[closed issues]:https://github.com/checkstyle/checkstyle/actions/workflows/no-old-refs.yml
```

Comment:
We forgot this and `run-link-check` in last PR referencing this issue. Now that I am looking, maybe it was better to separate them in a `minor: ..` commit.

Reply:
Done.